db/schema_tables.cc

```diff
@@ -3134,6 +3161,9 @@ std::vector<schema_ptr> all_tables(schema_features features) {
     if (features.contains<schema_feature::COMPUTED_COLUMNS>()) {
         result.emplace_back(computed_columns());
     }
+    if (features.contains<schema_feature::SCYLLA_KEYSPACES>()) {
+        result.emplace_back(scylla_keyspaces());
+    }
```

Comment:
Please check if something needs to be done to get Raft to manage this table when enabled. @gleb-cloudius @kbr-

Reply:
Still missing, no?

Reply:
Yes, I guess I'll reach out to one of the Raft people directly

Reply:
Kamil told me that as long as I add more mutations and use the existing announcing code, which I'm pretty sure I do, it will be handled by Raft correctly. I also verified by enabling Raft locally and observing that CREATE KEYSPACE and ALTER KEYSPACE statements with storage options specified are correctly recorded in group0_history table.

Reply:
But is the new table managed by raft?

Reply:
Ah, I understand the answer now.

@kbr- @kostja we should adjust DML so that writes into these tables go through the leader and we don't risk inconsistency.